modules/ossm-migrating-multitenant-with-cert-manager.adoc

```diff
@@ -0,0 +1,224 @@
+
+You can follow these same steps with your own workloads.
+
+. Update the `istio-csr` deployment to include your {SMProduct} 3 control plane by running the following command:
++
+[source,terminal]
+----
+  helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+      --install \
+      --reuse-values \
+      --namespace istio-system \
+      --wait \
+      -set "app.istio.revisions={basic,istio-tenant-a}" <1>
+----
++
+<1> The `app.istio.revisions` field needs to include your {SMProduct} 3.0 control plane revision _before_ you create your `Istio` resource so that proxies can properly communicate with the {SMProduct} 3.0 control plane.
+
+. Create your `Istio` resource.
++
+.Example `Istio` resource with cert-manager
++
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: istio-tenant-a
+spec:
+  namespace: istio-system-tenant-a <1>
+  version: v1.24.3
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            tenant: tenant-a <2>
+    #  extensionProviders:  <3>
+    #    - name: prometheus
+    #      prometheus: {}
+    #    - name: otel
+    #      opentelemetry:
+    #       port: 4317
+    #      service: otel-collector.opentelemetrycollector-3.svc.cluster.local
+  global:
+    caAddress: cert-manager-istio-csr.istio-system.svc:443
+    pilot:
+      env:
+        ENABLE_CA_SERVER: "false"
+----
++
+<1> The `spec.namespace` field in your `Istio` resource must be the **same** namespace as your `ServiceMeshControlPlane`. If you set the `spec.namespace` field in your `Istio` resource to a different namespace than your `ServiceMeshControlPlane`, the migration will not work properly.
+<2> controlplanes by default watch the entire cluster and when managing multiple controlplanes on a single cluster, you must narrow the scope of each controlplane by setting `discoverySelectors`. In this example, we use the label `tenant` but you can use any label or combination of labels that you choose.
+<3> If you are migrating metrics and tracing, uncomment and update the `extensionProviders` fields according to your tracing and metrics configurations.
+
+. Add your `tenant` label to each one of your dataplane namespaces by running the following command for each dataplane namespace:
++
+[source,terminal]
+----
+$ oc label ns bookinfo tenant=tenant-a
+----
++
+[NOTE]
+====
+In {SMProduct} 2.6, namespaces were enrolled into the mesh by adding them to the `Service MeshMemberRoll` resource. In {SMProduct} 3, you must label each one of your dataplane namespaces with the `discoverySelectors.matchLabels` label.
+====
+
+== Migrating workloads
+
+Now you can migrate your workloads from the {SMProduct} 2.6 controlplane to the {SMproduct} 3.0 controlplane.
+
+[NOTE]
+====
+You can migrate workloads and gateways separately.
```

Comment:
Same comment here about mentioning that you can also migrate them "in any order"